### ISSUE TITLE
flywire_ntpred() gives ids in numbers, not characters

### ISSUE THREAD

**Issue by YijieYin** [natverse/fafbseg#107]
`flywire_ntpred()` outputs ids in numbers. This isn't consistent with root ids that typically appears in characters in other functions, and thus may cause confusion when comparing. 
``` r
library(fafbseg)

id <- flywire_latestid('720575940603560294')
syn <- flywire_ntpred(id) 

typeof(syn$post_id)
#> [1] "double"
```

**Comment by jefferis** [comment]
Thanks @YijieYin. I see the issue, but this is actually somewhat complicated. flywire ids are 64 bit integers. Following your example you get:

```
library(fafbseg)
id <- flywire_latestid('720575940603560294')
syn <- flywire_ntpred(id) 

> class(syn$post_id)
[1] "integer64"
```
The problem is that R does not have a native 64 bit integer type, so we often need to accept or return the 64 bit integers as strings. However strings take up considerably more space 

```
id100=bit64::as.integer64('720575940603560294')+1:100
id100.st

> object.size(id100)
1080 bytes
> object.size(id100.str)
8848 bytes
```
Furthermore the neurotransmitter database stores the ids as integer64 internally.

My general intent is to return a character vector when we are dealing with whole neurons and integer64 when we are dealing with 1 row per synapse/connection. `flywire_ntpred` returns `integer64` ids as, does `flywire_partners` whereas `flywire_partner_summary` returns character.

So at least until some better solution suggests itself, I would suggest what is required is improved documentation /  examples. You can make a suggested change to the documentation by clcking on the `Source` link here:
![image](https://user-images.githubusercontent.com/23763/111042405-23948080-8435-11eb-9b92-5af64eff8588.png)

and then clicking on the pencil icon at https://github.com/natverse/fafbseg/blob/master/R/autosyn.R:

![image](https://user-images.githubusercontent.com/23763/111042490-8ede5280-8435-11eb-862d-29386f74f9f6.png)

**Comment by jefferis** [comment]
closed by #108
